src/Builtins.jl

```diff
@@ -19,21 +19,39 @@ end
 `@bind x Slider(1:10; default=8, show_value=true)`
 
 """
-Slider(range::AbstractRange; default=missing, show_value=false) = Slider(range, (default === missing) ? first(range) : default, show_value)
-
-function show(io::IO, ::MIME"text/html", slider::Slider)
-    print(io, """<input 
-        type="range" 
-        min="$(first(slider.range))" 
-        step="$(step(slider.range))" 
-        max="$(last(slider.range))" 
-        value="$(slider.default)"
-        $(slider.show_value ? "oninput=\"this.nextElementSibling.value=this.value\"" : "")
-        >""")
-    
-    if slider.show_value
-        print(io, """<output>$(slider.default)</output>""")
-    end
+Slider(range::AbstractRange; default=first(range), show_value=false) = Slider(range, default, show_value)
+
+function show(io::IO, mimetype::MIME"text/html", slider::Slider)
+    range, default, show_value = slider.range, slider.default, slider.show_value
+    show(io, mimetype, @htl("""
+    <span>
+        <input type="range" min="$(first(range))" max="$(last(range))" step="$(step(range))" value="$(default)">
+        $(HTML(show_value ? "<output>$(default)</output>" : ""))
+        
+        <script>
+            let parentnode = currentScript.parentElement
+            let slider = parentnode.querySelector("input")
+        
+            slider.addEventListener("input", e => {
+                parentnode.value = +slider.value
+                $(JavaScript(show_value ? "parentnode.querySelector(\"output\").value = +slider.value" : ""))
```

Comment:
Instead of generating JS like this, can you add line 34:
```julia
let show_value = $(show_value)
```

and then use normal JS in this line?

Reply:
The reason for I feel that doing so is superior is two-fold:

1. This minimizes the javascript on the page being rendered. When one attempts to study the Pluto cells, it is easier to understand if we provide less code in the first place, rather than having a bunch of objects in the page and hiding their visibility on the browser. Here, we minimized the code sent between the server and the client.
2. The pedagogical advantage of showing an example where we interpolate javascript code using Julia is very useful in my opinion.